sass/styles.scss

```diff
@@ -1,5 +1,8 @@
 @import "var";
 @import "tools";
 @import "base";
-@import "layout";
-@import "module";
+@import "Modules/buttom-see-more";
```

Comment:
I like how you are dividing this. 
Now please adjust the markup and the css in order to follow the BEM methodology we discussed together.